modules/customize-installation-operator-overlays.adoc

```diff
@@ -39,21 +39,22 @@ The following example shows the structure of an overlay:
 [source,yaml]
 ----
 overlays:
-- apiVersion: v1     # <1>
-  kind: ConfigMap    # <2>
-  name: my-configmap # <3>
+- apiVersion: v1         # <1>
+  kind: ExampleKind      # <2>
+  name: my-resource      # <3>
   patches:
-    - path: .data    # <4>
-      value: |       # <5>
+    - path: .some.field  # <4>
+      value: |           # <5>
         key1: data2
         key2: data2
 ----
 
 <1> Targeted Kubernetes resource ApiVersion, for example `apps/v1`, `v1`, `networking.k8s.io/v1`
 <2> Resource type (e.g., Deployment, ConfigMap, NetworkPolicy)
-<3> Name of the resource, for example `my-configmap`
+<3> Name of the resource, for example `my-resource`
 <4> JSONPath expression to the field, for example `spec.template.spec.containers[name:central].env[-1]`
-<5> YAML string for the new field value
+<5> YAML string for the new field value. If YAML parsing is not desired, an alternative is to use the `verbatim` key,
```

Comment:
```suggestion
<5> YAML string for the new field value. If you do not want to use YAML parsing, you can use the `verbatim` key as shown in the following ConfigMap example.
```
IBM Style has us use "wanted" instead of "desire".